docs/source/mcmc_samplers/index.rst

```diff
@@ -22,4 +22,5 @@ interface, that can be used to sample from an unknown
     metropolis_mcmc
     population_mcmc
     slice_doubling_mcmc
-    slice_stepout_mcmc
\ No newline at end of file
+    slice_stepout_mcmc
+    slice_covariance_matching
```

Comment:
Can you move slice_covariance_matching up so that the list is alphabetical? Sorry!